### PR TITLE
use stable release of willdurand/negotiation

### PR DESCRIPTION
**Pull request by xabbuh**
The stable release [is now a thing](https://github.com/willdurand/Negotiation/releases/tag/v2.0.0).

### CODE REVIEW

**Comment by willdurand** [comment]
:+1:

File: composer.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
         "symfony/framework-bundle":       "^2.7",
         "symfony/routing":                "^2.7",
         "doctrine/inflector":             "^1.0",
-        "willdurand/negotiation":         "^2.0.0-alpha1",
+        "willdurand/negotiation":         "^2.0.0",
```
Comment (Soullivaneuh) [inline composer.json]: Should be `^2.0`.

Comment (xabbuh) [inline composer.json]: we can do that but it does not make a difference

Comment (Soullivaneuh) [inline composer.json]: Indeed, forgot the caret rule.

But it's still better to see. :-)

Comment (lsmith77) [inline composer.json]: yeah can you fix this minor detail to improve readability?

Comment (xabbuh) [inline composer.json]: done

**Pull request merged**